code/modules/mob/living/carbon/alien/humanoid/death.dm

```diff
@@ -19,3 +19,13 @@
 		mind.store_memory("Time of death: [tod]", 0)
 
 	return ..(gibbed)
+
+//When the alien queen dies, all others must pay the price for letting her die.
+/mob/living/carbon/alien/humanoid/royal/queen/death(gibbed)
+	if(stat == DEAD)
+		return
+
+	for(var/mob/living/carbon/alien/humanoid/A in living_mob_list)

```

Comment:
Only humanoid aliens? What about larva? Wouldn't it be possible for a larva to evolve, then be fastracked to queen via plasma transfers from other aliens to bypass this?